components/automate-ui/src/app/entities/service-groups/service-groups.actions.ts

```diff
@@ -21,7 +21,11 @@ export enum ServiceGroupsActionTypes {
 
   GET_SERVICES_BY_SERVICE_GROUP = 'SERVICE_GROUPS::SERVICES::GET',
   GET_SERVICES_BY_SERVICE_GROUP_SUCCESS = 'SERVICE_GROUPS::SERVICES::GET::SUCCESS',
-  GET_SERVICES_BY_SERVICE_GROUP_FAILURE = 'SERVICE_GROUPS::SERVICES::GET::FAILURE'
+  GET_SERVICES_BY_SERVICE_GROUP_FAILURE = 'SERVICE_GROUPS::SERVICES::GET::FAILURE',
+
+  GET_NODE_SUGGESTIONS = 'GET_NODE_SUGGESTIONS::SERVICES::GET',
+  GET_NODE_SUGGESTIONS_SUCCESS = 'GET_NODE_SUGGESTIONS::SERVICES::SUCCESS',
+  GET_NODE_SUGGESTIONS_FAILURE = 'GET_NODE_SUGGESTIONS:SERVICES::FAILURE'
```

Comment:
Should we rename these actions? or why do we have the word "node" in the applications tab? 🤔

Reply:
@afiune Updated name `GET_NODE_SUGGESTIONS` to be specific to service groups: `GET_SERVICE_GROUP_SUGGESTIONS`